mteb/abstasks/AbsTaskBitextMining.py

```diff
@@ -25,49 +25,36 @@ def evaluate(self, model, split, **kwargs):
         if not self.data_loaded:
             self.load_data()
 
+        parallel_subsets = (
+            self.parallel_subsets if hasattr(self, "parallel_subsets") else False
+        )
         if self.is_crosslingual:
-            scores = {}
-            for lang in self.dataset:
-                logger.info(
-                    f"\nTask: {self.metadata_dict['name']}, split: {split}, language: {lang}. Running..."
-                )
-                data_split = self.dataset[lang][split]
-                scores[lang] = self._evaluate_split(model, data_split, **kwargs)
+            if parallel_subsets:
+                scores = self._evaluate_split(model, self.dataset[split], **kwargs)
+            else:
+                scores = {}
+                for lang in self.dataset:
+                    logger.info(
+                        f"\nTask: {self.metadata_dict['name']}, split: {split}, language: {lang}. Running..."
+                    )
+                    data_split = self.dataset[lang][split]
+                    scores[lang] = self._evaluate_split(
+                        model, data_split, subsets=["sentence1", "sentence2"], **kwargs
+                    )
         else:
             logger.info(
                 f"\nTask: {self.metadata_dict['name']}, split: {split}. Running..."
             )
             data_split = self.dataset[split]
-            scores = self._evaluate_split(model, data_split, **kwargs)
+            print(data_split)
+            scores = self._evaluate_split(
+                model, data_split, subsets=["sentence1", "sentence2"], **kwargs
+            )
 
         return scores
 
     def _evaluate_split(self, model, data_split, **kwargs):
-        if len(data_split["sentence1"]) == 1:
-            sentence1 = data_split["sentence1"][0]
-        else:
-            sentence1 = data_split["sentence1"]
-        if len(data_split["sentence2"]) == 1:
-            sentence2 = data_split["sentence2"][0]
-        else:
-            sentence2 = data_split["sentence2"]
-
-        if "gold" not in data_split.features:
-            assert len(sentence1) == len(sentence2), "Wrong dataset format"
-            n = len(sentence1)
-            gold = list(zip(range(n), range(n)))
-        else:
```

Comment:
I removed the 'gold' loading option, as only BUCC was using it, and converted BUCC accordingly
As a result, the results on BUCC dataset are not backward compatible and they have slightly increased
Tbh the results were previously already very high, so I'm not sure this dataset is very useful

Reply:
What change in score are we talking about here?

Reply:
> What change in score are we talking about here?

You can see the difference in the BUCC.json file, but for example for the de-en split:

"de-en": {
-      "accuracy": 0.9736951983298539,
+      "accuracy": 0.9847599164926931,
-      "f1": 0.9711203897007653,
+      "f1": 0.9798434237995824,
-      "main_score": 0.9711203897007653,
+      "main_score": 0.9798434237995824,
-      "precision": 0.9698329853862213,
+      "precision": 0.9774269311064718,
+      "recall": 0.9847599164926931
-      "recall": 0.9736951983298539
}

There's like a 1% point increase in scores, the scores are very high though